.circleci/config.yml

```diff
@@ -692,20 +862,29 @@ workflows:
       - unittest-35
       - unittest-27
       - unittest-26
+      - smoke-rpm-package-py3
```

Comment:
We have tons of job now so we will need to keep an eye on our Circle CI usage and potentially adjust which jobs run on every push and which only on merge to master or periodically.